crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -2783,6 +2783,40 @@
         }
       ]
     },
+    {
+      "name": "sui_tryGetObjectBeforeVersion",
+      "tags": [
+        {
+          "name": "Read API"
+        }
+      ],
+      "description": "Note there is no software-level guarantee/SLA that objects with past versions can be retrieved by this API, even if the object and version exists/existed. The result may vary across nodes depending on their pruning policies. Returns the latest object information with a version less than or equal to the given version",
+      "params": [
+        {
+          "name": "object_id",
+          "description": "the ID of the queried object",
+          "required": true,
+          "schema": {
+            "$ref": "#/components/schemas/ObjectID"
+          }
+        },
+        {
+          "name": "version",
+          "description": "the version of the queried object",
+          "required": true,
+          "schema": {
+            "$ref": "#/components/schemas/SequenceNumber"
+          }
+        }
+      ],
+      "result": {
+        "name": "SuiPastObjectResponse",
+        "required": true,
+        "schema": {
+          "$ref": "#/components/schemas/ObjectRead"
+        }
+      }
```

Comment:
Do you know if we have a way to mark the API as hidden so that spec information isn't generated for it?

Reply:
added functionality so methods that are marked as deprecated won't be displayed